launch_test.go

```diff
@@ -40,27 +40,6 @@ type MockRepo struct {
 	path     func() string
 }
 
-func (r MockRepo) Checkout() error {
-	if r.checkout != nil {
-		return r.checkout()
-	}
-	return nil
-}
-
-func (r MockRepo) MergePR(prNumber, sha string) error {
-	if r.mergePR != nil {
-		return r.mergePR(prNumber, sha)
-	}
-	return nil
-}
-
-func (r MockRepo) Path() string {
-	if r.path != nil {
-		return r.path()
-	}
-	return ""
-}
-
 func mockAPI(t *testing.T, testBuildID, testJobID, testPipelineID string, testStatus screwdriver.BuildStatus) MockAPI {
```

Comment:
Should also remove https://github.com/screwdriver-cd/launcher/blob/79990efb37df711f02fda7a3424de44c802258f4/launch_test.go#L37